tests/ui/transmutability/enums/repr/primitive_reprs_should_have_correct_length.stderr

```diff
@@ -90,7 +90,7 @@ error[E0277]: `u8` cannot be safely transmuted into `V0i16` in the defining scop
   --> $DIR/primitive_reprs_should_have_correct_length.rs:72:44
    |
 LL |         assert::is_transmutable::<Smaller, Current, Context>();
-   |                                            ^^^^^^^ At least one value of `u8` isn't a bit-valid value of `V0i16`
```

Comment:
The changes in this file are to be expected, I made it so that `DstIsBitIncompatible` is only used if there are no other errors that are more specific